{
    "id": 255532,
    "name": "CVE-2024-39428",
    "cwe": "Unknown",
    "modified": "2024-07-01T10:37:24",
    "published": "2024-07-01T07:15:06",
    "status": "Awaiting Analysis",
    "summary": "In trusty service, there is a possible out of bounds write due to a missing bounds check. This could lead to local denial of service with System execution privileges needed",
    "cvss3": {
        "cvss": 6.8,
        "exploitability_score": 2.5,
        "impact_score": 4.2,
        "vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:N/I:L/A:H"
    },
    "cvss2": null,
    "cpes": []
}